README.md

```diff
@@ -17,8 +17,11 @@ This homework is for understanding a concept of Robot Architecture, and ROS(Robo
 Before doing this homework, you have to setup catkin ROS workspace for executing homework.
 
 ```bash
-# install catkin tools
+# install catkin tools (except Ubuntu 14.04)
 $ sudo apt-get install python-catkin-tools python-rosdep ros-hydro-desktop-full
```

Comment:
I think this is better to be
`sudo apt-get install python-catkin-tools python-rosdep ros-indigo-desktop-full`